Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Ride/EndRide.hs

```diff
@@ -409,9 +409,10 @@ endRide handle@ServiceHandle {..} rideId req = withLogTag ("rideId-" <> rideId.g
     fork "DriverRideCompletedCoin Event : " $ do
       expirationPeriod <- DC.getExpirationSeconds thresholdConfig.timeDiffFromUtc
       validRideTaken <- DC.checkHasTakenValidRide (Just chargeableDistance)
+      metroRide <- checkSplLocation booking.specialLocationTag "SureMetro"
```

Comment:
https://github.com/nammayatri/nammayatri/commit/789bfe09a81126c083eab505fc7ba37087408558

Recently one change is Merged in Frontend where if Backend we add `SureWarriorMetro` as a category in specialLocationTag then also it means metro. So we can do that handling here to avoid any conflicts once that change is done at backend.

`metroRide <- checkSplLocation booking.specialLocationTag "SureMetro" || checkSplLocation booking.specialLocationTag "SureWarriorMetro"`